src/EnergyPlus/DXCoils.cc

```diff
@@ -5536,9 +5536,9 @@ void GetDXCoils(EnergyPlusData &state)
         } else {
             {
                 if (state.dataCurveManager->PerfCurve(indexSHCurve).ObjectType == "Curve:Quadratic") {
```

Comment:
You should probably be testing `curveType` rather than `objectType` here.

Reply:
:+1:

Reply:
I just went ahead and removed the objectType member entirely, added a small constexpr std array for those rare cases where we need to print the object type in an error message, and moved all comparisons to the enum curveType.